index.md

```diff
@@ -499,6 +499,52 @@ LO: Students should be able to...
 6. How does this project test itself?
 7. How does this project do CI, if at all?
 
+## Academic Integrity
+
+You are expected to adhere to [Tufts' Academic Integrity Policy][1] in this
+course. Plagarism of code or answers to assignments and sharing answers with or
+accepting answers from others is strictly prohibited.
+
+You are allowed to reference Linux man pages and any open source projects while
+completing assignments. You are also allowed to read any internet resource,
+with the exception of material outside this website that explicitly pertains to
+this course (e.g. a former student's GitHub repo with assignment solutions).
```

Comment:
Consider also banning people from publishing such repos

Reply:
Think this is covered in the opening paragraph: "... sharing answers with ...".

Reply:
Might be worth being explicit